identifiers/country-ar/ar_first_level_ocdid.csv

```diff
@@ -0,0 +1,25 @@
+id,name
+ocd-division/country:ar/autonomous_city:df,Buenos Aires City
```

Comment:
it looks like these are these are the HASC codes for these administrative areas. The ISO codes are single letter in Argentina, so for Buenos Aires City it's `C` and fro Buenos Aires it's `B`

Reply:
Done.